pkg/util/capabilities/config_test.go

```diff
@@ -75,25 +70,6 @@ func TestReadFromWriteTo(t *testing.T) {
 
 }
 
-func checkReadFrom(t *testing.T, confExpected Config, shouldFail bool) {
```

Comment:
Why are we deleting this here?

Reply:
Nothing is using the function. :man_shrugging: